config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+  kubernetes-sigs/kube-storage-version-migrator:
+  - name: pull-migrator-test
```

Comment:
this looks like a good candidate for https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md which are replacing bootstrap.py (slowly!)

basically just:
- add `decorate: true` below the `name: pull-migrator-test`
- add a `path_alias: my.go/import-path` below that if your go import path is not github.com/kubernetes-sigs/kube-storage-version
- drop the label / preset-service-account
- on the container below, replace the args with:
```
command:
- make
- test
```

Reply:
Done. Can you take another look? Thanks.